urnx/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 The Infratographer Authors
```

Comment:
Does it matter that it's 2023 now?

Reply:
I'm going to leave this as 2022 for now (since it lines up with the rest). I'm assuming at some point we'll put together a workflow that goes through and ensures all the licenses are lined up.

Reply:
Works for me!